frontend/src/common/queries/wheresMyGene.ts

```diff
@@ -370,9 +358,11 @@ export function useWMGQuery(
 
   return useQuery(
     [USE_QUERY, query, currentSnapshotId],
-    ({ signal }) => fetchQuery({ query, signal, version }),
+    ({ signal }) => fetchQuery({ query, signal }),
     {
+      // (seve): my understanding of this flag is that we only want to automatically refetch on snapshotID change if the query is not null
       enabled: Boolean(query),
+      keepPreviousData: true,
```

Comment:
new flag!

Reply:
Let's add a comment and link to the doc! Such a useful feature 🔥 Great find!